calicoctl/calicoctl/commands/convert.go

```diff
@@ -239,3 +258,56 @@ func getTypeConverter(resKind string) (converters.Converter, error) {
 		return nil, fmt.Errorf("conversion for the resource type '%s' is not supported", resKind)
 	}
 }
+
+func unpackResourceLists(convRes []unversioned.Resource) ([]unversioned.Resource, error) {
+	var unpackedConvRes []unversioned.Resource
+	for i, convResource := range convRes {
+		if strings.EqualFold(convResource.GetTypeMetadata().Kind, resourceloader.KindK8sListV1) && strings.EqualFold(convResource.GetTypeMetadata().APIVersion, resourceloader.VersionK8sListV1) {
+			// Remove list from convRes
```

Comment:
I don't think you actually need to remove the item from the list, right? Since we're just iterating over this once and building a completely separate list below.

Reply:
Oh right, I think that's leftover from when I was trying to do this in place (I later refactored it into a separate function). Will fix!